dist/osx/build_osx_package.sh

```diff
@@ -22,6 +22,9 @@ cd "${RIZININSTALL}" && find . | cpio -o --format odc | gzip -c > "${OSXPKGDIR}/
 mkbom "${RIZININSTALL}" "${OSXPKGDIR}/Bom"
 pkgutil --flatten "${OSXPKGDIR}" "${RIZINDIR}/dist/osx/rizin.pkg"
 
+# convert -size 620x418 xc:none -background none /tmp/bg.png
+# composite -geometry +30+230 -background none \( "${RIZINDIR}/doc/img/rizin.svg" -resize 25% \) /tmp/bg.png  -colorspace sRGB "${RIZINDIR}/dist/osx/Resources/rizin-logo.png"
```

Comment:
Any reason to have it in the yml and not here? I think it fits better into this script to have everything self-contained.

Reply:
It'll introduce dependency on `imagemagick`. That needs to be documented as well, but I couldn't find a good place.

Reply:
You could do something like `convert ... || { echo convert failed, is imagemagick installed?; exit 1; }`
Then one will see right away what is wrong.

Reply:
We can probably remove the current `rizin-logo.png` completely now?

Reply:
Yes (as you have already done)